app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,4 @@
 <resources>
     <string name="app_name">ARK Navigator</string>
+    <string name="as_is_sorting_selected">This option will be applied when the screen is re-opened.</string>
```

Comment:
Don't want to be picky, but I think dots at the end are not necessary in one-line messages.
Also, maybe "This option will be applied during next resources selection" would be more semantically correct.